supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -690,21 +691,21 @@ There is no functional difference between the first server that was installed an
 [discrete]
 [[itanium]]
 ==== image:images/yes.png[yes] Itanium (noun)
-*Description*: "Itanium" is a 64-bit RISC microprocessor and a member of Intel's Merced family of processors. Based on the Explicitly Parallel Instruction Computing (EPIC) design philosophy, which states that the compiler should decide which instructions be executed together, Itanium has the highest FPU power available. In 64-bit mode, Itanium is able to calculate two bundles of a maximum of three instructions at a time. In 32-bit mode, it is much slower. Decoders must first translate 32-bit instruction sets into 64-bit instruction sets, which results in extra-clock cycle use. Itanium's primary use is driving large applications that require more than 4 GB of memory, such as databases, ERP, and future internet applications.
+*Description*: _Itanium_ is a 64-bit RISC microprocessor and a member of Intel's Merced family of processors. Based on the Explicitly Parallel Instruction Computing (EPIC) design philosophy, which states that the compiler should decide which instructions be executed together, Itanium has the highest FPU power available. In 64-bit mode, Itanium is able to calculate two bundles of a maximum of three instructions at a time. In 32-bit mode, it is much slower. Decoders must first translate 32-bit instruction sets into 64-bit instruction sets, which results in extra-clock cycle use. Itanium's primary use is driving large applications that require more than 4 GB of memory, such as databases, ERP, and future internet applications.
```

Comment:
Since this just came up on the style council chat group, I notice there's a `'s` after Intel, which is a brand name so I think shouldn't be punctuated. I'm not sure about `Itanium's` but would err on the side of removing the `'s`in case.
We could possibly say: 

`..a member of the Intel Merced family of processors. `
and 
`The primary use of Itanium microprocessors is to drive large applications ...`